frontend/src/pages/grants/reply/index.tsx

```diff
@@ -217,19 +221,9 @@ const GrantReply = () => {
             <FormattedMessage
               id="grants.reply.messageDescription"
               values={{
-                visaApplicationFormLink: (
-                  <Link
-                    target="_blank"
-                    href={data.conference.visaApplicationFormLink}
-                  >
-                    <Text
-                      decoration="underline"
-                      size={2}
-                      weight="strong"
-                      color="none"
-                    >
-                      <FormattedMessage id="grants.reply.visaApplicationFormLink" />
-                    </Text>
+                visaPageLink: (
+                  <Link target="_blank" href={visaPageLink}>
+                    {`https://pycon.it${visaPageLink}`}
```

Comment:
why this?

Reply:
should I just hardcoed? it? 
```suggestion
                    {`https://pycon.it/visa/`}
```

Reply:
Because in the sentence think is more clear if we show the link?

> `For Visa invitation letter requests, kindly visit our dedicated page at {visaPageLink} and submit the form there.
We'll process your application and send the invitation letter within 4 days of form submission.`

Reply:
It is an internal page on the website, so the text could just say:

> For Visa invitation letter requests, kindly visit our dedicated page VISA page and submit the form there. 

Where "VISA page" is a clickable link

Also I don't think we should say "We'll process your application and send the invitation letter within 4 days of form submission." in here, as if things change (how long it takes to process an application) we would need now to change 2 places. I think it should just say to go to the visa page